holoviews/core/data/iris.py

```diff
@@ -232,7 +253,7 @@ def select_to_constraint(cls, selection):
             if isinstance(constraint, slice):
                 constraint = (constraint.start, constraint.stop)
             if isinstance(constraint, tuple):
-                constraint = iris.util.between(*constraint)
+                constraint = iris.util.between(*constraint, rh_inclusive=False)
```

Comment:
Yes, so far we've used non-inclusive upper bounds in holoviews, so this just makes it consistent. Might be worth having the discussion which is more appropriate, since iris and xarray both seem to use inclusive bounds.